source/fundamentals/connection/connection-options.txt

```diff
@@ -21,51 +21,69 @@ parameters of the connection URI to specify the behavior of the client.
      - Specifies the minimum number of connections that must exist at
        any moment in a single connection pool.
 
+       | **Default**: ``0``
+
    * - **maxPoolSize**
      - integer
      - Specifies the maximum number of connections that a connection
        pool may have at a given time.
 
+       | **Default**: ``100``
+
    * - **waitQueueTimeoutMS**
      - integer
      - Specifies the maximum amount of time, in milliseconds that a
        thread may wait for a connection to become available.
 
+       | **Default**: ``120000`` (2 minutes)
```

Comment:
issue: for consistency and because we aren't stating minutes in the explanation, let's replace 2 minutes with 120 seconds

```suggestion
       | **Default**: ``120000`` (120 seconds)
```